### ISSUE TITLE
tfrecords files are multiple times larger than the raw image data

### ISSUE THREAD

**Issue by HtutLynn** [MaybeShewill-CV/CRNN_Tensorflow#374]
Hi, I am implementing an OCR system for my language using your code base as a reference.
My `train_images_size` is around 18 GB and `test` and `val` folders each having around 4GB. I used `write_tfrecords.py` function to generate tfrecords from those data using 4 threads(P.S I have 4-core CPU so).  train_images_size exploded into 4 tfrecords file with around 45 GB each. Is it normal or am I doing something wrong?
![Screenshot from 2019-12-18 20-46-38](https://user-images.githubusercontent.com/42746452/71093431-92097500-21d7-11ea-9d4a-b6df4776ce57.png)

**Comment by MaybeShewill-CV** [comment]
@HtutLynn normal:)

**Comment by HtutLynn** [comment]
@MaybeShewill-CV Alright, thanks. I made some modifications to `write_tfrecords.py`. Instead of writing the raw numpy array into tfrecords, I decoded those numpy data, obtained from cv2.imread by using cv2.imdecode func. It significantly reduced the storage that generated tfrecords take.
![Screenshot from 2019-12-18 23-01-45](https://user-images.githubusercontent.com/42746452/71104595-6f348c00-21ea-11ea-8aaf-7d9838e7eb1a.png)

Btw, how long did it take to train the model to converge on your system? I have around 5 millions image data and planning to train on GTX 1070.
train_image : `32` x 960
seq_length : `120`
sample image: 
![sample](https://user-images.githubusercontent.com/42746452/71104742-ac991980-21ea-11ea-9176-3459f89d1b36.jpg)
